components/CreateOrganizationForm.js

```diff
@@ -272,7 +271,6 @@ const CreateOrganizationForm = props => {
                         labelColor="black.700"
                         labelFontWeight="600"
                         value={values.website}
-                        required
```

Comment:
Rather than just removing it, we can do `required={false}`, which adds `(Optional)` in the label